### PR TITLE
add new admin and send mail when document upload

### PR DESCRIPTION
**Pull request by vannam1996**

### CODE REVIEW

**Comment by vannam1996** [comment]
image

**Comment by vannam1996** [comment]
ready

File: config/environments/development.rb [hunk 1]
```diff
@@ -1,54 +1,43 @@
...
 
-  # Raises error for missing translations
-  # config.action_view.raise_on_missing_translations = true
-
-  # Use an evented file watcher to asynchronously detect changes in source code,
-  # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+   :address              => "smtp.gmail.com",
```
Comment (dothidiemthao) [inline config/environments/development.rb]: ko dùng : =>

**Comment by dothidiemthao** [comment]
commented

**Comment by vannam1996** [comment]
ready

**Pull request merged**